<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dimensao</class>
 <widget class="QDialog" name="Dimensao">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>197</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>12</y>
     <width>170</width>
     <height>118</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider_DimensaoX">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumber_DimensaoX"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider_DimensaoY">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumber_DimensaoY"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Z</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider_DimensaoZ">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumber_DimensaoZ"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_NovaEscultura">
      <property name="text">
       <string>Nova Escultura</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider_DimensaoZ</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcdNumber_DimensaoZ</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_DimensaoY</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcdNumber_DimensaoY</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_DimensaoX</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcdNumber_DimensaoX</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
